### PR TITLE
Added openjdk11 and other improvements

### PR DESCRIPTION
**Pull request by arctan5x**

### CODE REVIEW

File: roles/splunk_common/tasks/java_tasks/install_oracle8_jdk.yml [hunk 1]
```diff
@@ -57,3 +57,34 @@
...
+  when:
+    - ansible_distribution == 'Debian'
+
+- name: Set JAVA_HOME in splunk-launch.conf
+  become: yes
+  become_method: sudo
+  become_user: root
+  lineinfile:
+    path: "{{splunk.home}}/etc/splunk-launch.conf"
+    regexp: '^JAVA_HOME'
```
Comment (nwang92) [inline roles/splunk_common/tasks/java_tasks/install_oracle8_jdk.yml]: If there's no regex match, will it still write this line?

Comment (arctan5x) [reply roles/splunk_common/tasks/java_tasks/install_oracle8_jdk.yml]: yes

File: inventory/environ.py [hunk 2]
```diff
@@ -104,13 +104,24 @@ def getDefaultVars():
...
-            defaultVars["java_update_version"] = re.search("jdk-8u(\d+)-linux-x64.tar.gz", defaultVars["java_oracle_url"]).group(1)
-        except:
-            raise Exception("Invalid Java download URL format")
+        if java_version == "oracle:8":
+            defaultVars["java_download_url"] = os.environ.get("JAVA_DOWNLOAD_URL", "https://download.oracle.com/otn-pub/java/jdk/8u201-b09/42970487e3af4f5aa5bca3f542482c60/jdk-8u201-linux-x64.tar.gz")
+            try:
+                defaultVars["java_update_version"] = re.search("jdk-8u(\d+)-linux-x64.tar.gz", defaultVars["java_download_url"]).group(1)
+            except:
+                raise Exception("Invalid Java download URL format")
+        elif java_version == "openjdk:11":
```
Comment (jmeixensperger) [inline inventory/environ.py]: What about openjdk:8? Do we not need to set java_download_url or java_update_version?

Comment (arctan5x) [reply inventory/environ.py]: For openjdk, we install it through the package manager so no need for the url.

**Pull request merged**